### ISSUE TITLE
Loss function call mistake found - calling sample() with normalized configuration

### ISSUE THREAD

**Issue by 0xWILL225** [fishbotics/avoid-everything#7]
It appears that TorchFrankaSampler.sample() is being called with a normalized configuration in loss.py, which in turn calls TorchURDF.visual_geometry_fk_batch(), which to the best of my knowledge expects non-normalized configurations as input. It seems to me that this should not work --- I don't believe the training would converge even with all the bugs fixed that makes the current code in this repo not run.

Please, I would very much appreciate it if you would release a working version of the code, exactly as it was used to reach the results presented in the paper. I am excited to replicate it.

Cheers,
William
